Source/UI/Areas/Api/Controllers/PlayedGamesController.cs

```diff
@@ -122,7 +122,7 @@ internal virtual HttpResponseMessage GetPlayedGameSearchResults(PlayedGameFilter
         {
             var filter = _transformer.Transform<PlayedGameFilter>(playedGameFilterMessage);
 
-            var searchResults = _playedGameRetriever.SearchPlayedGames(filter);
+            var searchResults = _playedGameRetriever.SearchPlayedGames(filter, null);

```

Comment:
I didn't have access to an `ApplicationUser` because this is the API. How should I go about handling this?